packages/integration-tests/src/EntityManager.clone.test.ts

```diff
@@ -176,6 +182,20 @@ describe("EntityManager.clone", () => {
     expect(a2.comments.get[0].parent.get).toBe(a2);
   });
 
+  it("can clone polymorphic references directly", async () => {
+    const em = newEntityManager();
+    // Given an entity with a polymorphic parent
+    const c1 = newComment(em, { parent: {} });
+    expect(c1.parent.get).toBeInstanceOf(Author);
+    await em.flush();
+    // When we clone it in a new UoW
+    const em2 = newEntityManager();
+    const c2 = await em2.load(Comment, c1.idOrFail);
+    const c3 = await em2.clone(c2);
+    // Then the parent can be loaded
+    expect(await c3.parent.load()).toBeInstanceOf(Author);
```

Comment:
This test was breaking until the "teach PolymorphicReference about having ids set" fix (given em.clones new pass-opts-to-the-cstr strategy).